README.md

```diff
@@ -18,6 +18,91 @@ Using the non-normalized data in the **BlogPost** table below, create a database
 
 Fork this repository and add your tables here:
 
+### BlogPost
+| BlogPostId | Slug                          | Title                                | Body           |
+|------------|-------------------------------|--------------------------------------|----------------|
+| 1          | serilog-structured-logging    | Structured Logging with Serilog      | Long Text Here |
+| 2          | persisting-smart-enum         | Persisting a Smart Enum with EF Core | Long Text Here |
+| 3          | localization-aspnet-core-apis | Localization in ASP.NET Core APIs    | Long Text Here |
+
+
+### Author
+| AuthorId | AuthorName    | AuthorEmail                 | AuthorBio      |
+|----------|---------------|-----------------------------|----------------|
+| 1        | Eric Fleming  | ericfleming@nimblepros.com  | Long Text Here |
+| 2        | Kyle McMaster | kylemcmaster@nimblepros.com | Long Text Here |
+| 3        | Steve Smith   | steve@nimblepros.com        | Long Text Here |
+
+
+### BlogPostAuthor
+| BlogPostId | AuthorId |
+|------------|----------|
+| 1          | 1        |
+| 1          | 2        |
+| 2          | 3        |
+| 3          | 2        |
+
+
+### Category
+| CategoryId | Category             |
+|------------|----------------------|
+| 1          | Software Development |
+| 2          | Cloud Computing      |
+| 3          | Web APIs             |
+
+
+### BlogPostCategory
+| BlogPostId | CategoryId |
+|------------|------------|
+| 1          | 1          |
+| 2          | 1          |
+| 3          | 1          |
+| 1          | 2          |
+| 3          | 3          |
+
+
+
+### Tag
+| TagId | Tag          |
+|-------|--------------|
+| 1     | logging      |
+| 2     | serilog      |
+| 3     | appinsights  |
+| 4     | efcore       |
+| 5     | smartenum    |
+| 6     | localization | 
+| 7     | aspnetcore   |
+| 8     | web-api      |
```

Comment:
Good. I like that the tags are all lowercase in the database. How would you deal with authors wanting their tags to retain their case, while still wanting to return articles based on a tag using a case-insensitive search?

Example: 
Article 1 is tagged AspNetCore (and renders using this casing)
Article 2 is tagged aspnetcore (and renders using this casing)
Search for articles tagged 'aspnetcore' should return both of them.

Reply:
SqlServer search is case-insensitive by default but if this has been changed I'd use a case-insensitive collation 
eg. select * from Tag where tag = 'searchstring' collate Latin1_General_100_CI_AS or in code, EF.Functions.Collate(tag, "Latin1_General_100_CI_AS"), assuming EF and SqlServer.
Alternatively, I'd convert both the searchstring and the tag to lowercase when doing the search.

Reply:
If SQL Server is always case-insensitive then when the first user adds a tag "AspNetCore" it will get id 7 or whatever. Then another user will try to add "aspnetcore" and you'll do a search to see if it already exists, and you'll come back with the ID 7 record, which doesn't match case. If the requirements are that each individual user and blog post can have and retain whatever style of casing they want (whatever they used when they wrote the article), what would you change? Presumably you'd just make sure the search for existing tags *was* case sensitive?

Reply:
Ah yes! Good point. I didn't consider that.  So for the insert, I'd make the search to check if the tag already exists case-sensitive.  When doing a search for posts by tag, then the search should be case-insensitive.

Reply:
Yep, sounds good!